### PR TITLE
Update AUTHORS for 1.5, ref #10451

### PR DESCRIPTION
**Pull request by nealmcb**
Signed-off-by: Neal McBurnett neal@mcburnett.org

### CODE REVIEW

**Comment by tiborvass** [comment]
@nealmcb it's out of date, missing Katie McLaughlin and Mihai Borobocea

**Comment by nealmcb** [comment]
Thanks, @tiborvass  - it's up-to-date now.

**Comment by tiborvass** [comment]
LGTM

Ping @tianon

File: AUTHORS [hunk 1]
```diff
@@ -62,6 +65,7 @@ Andy Rothfusz <github@developersupport.net>
 Andy Smith <github@anarkystic.com>
 Andy Wilson <wilson.andrew.j+github@gmail.com>
 Ankush Agarwal <ankushagarwal11@gmail.com>
+Ankush Agarwal <ankushagarwal@users.noreply.github.com>
```
Comment (tianon) [inline AUTHORS]: This one needs a new `.mailmap` entry. :smile:

File: AUTHORS [hunk 2]
```diff
@@ -202,6 +211,7 @@ Dominik Honnef <dominik@honnef.co>
 Don Kjer <don.kjer@gmail.com>
 Don Spaulding <donspauldingii@gmail.com>
 Doug Davis <dug@us.ibm.com>
+Doug Davis <duglin@users.noreply.github.com>
```
Comment (tianon) [inline AUTHORS]: Another `.mailmap` candidate. :+1:

File: AUTHORS [hunk 3]
```diff
@@ -653,6 +684,7 @@ Sjoerd Langkemper <sjoerd-github@linuxonly.nl>
 Solomon Hykes <solomon@docker.com>
 Song Gao <song@gao.io>
 Soulou <leo@unbekandt.eu>
+soulshake <aj@gandi.net>
 soulshake <amy@gandi.net>
```
Comment (tianon) [inline AUTHORS]: Another `.mailmap` candidate. :+1:

File: AUTHORS [hunk 4]
```diff
@@ -669,12 +702,15 @@ Sébastien <sebastien@yoozio.com>
 Sébastien Luttringer <seblu@seblu.net>
 Sébastien Stormacq <sebsto@users.noreply.github.com>
 tang0th <tang0th@gmx.com>
+Tangi COLIN <tangicolin@gmail.com>
+tangicolin <tangicolin@gmail.com>
```
Comment (tianon) [inline AUTHORS]: :+1:

**Comment by tianon** [comment]
Overall LGTM, but a few `.mailmap` entries need to be added.

**Comment by jfrazelle** [comment]
@tianon can that be done after the fact? is this good to merge? or just waiting on revisions?

**Comment by tianon** [comment]
IMO we shouldn't merge until `.mailmap` is updated -- otherwise, there's no point in even maintaining this file at all, and we might as well just tell people "run this Git one-liner to see a list of people who've worked on the project"

**Comment by jfrazelle** [comment]
ok ok we wait :D I just was confused by your LGTM sorry

**Comment by nealmcb** [comment]
Ahh - that makes it a bit of a new ballgame, with human intervention needed.  That should probably be noted in the relevant documentation (or discussed, or whatever).
So were does this fit into the 1.5.0 release?  If I take a shot at the .mailcap updates and update AUTHORS, would that make it in to the release?  By what deadline?

**Comment by nealmcb** [comment]
So I guess we might want the project/generate-authors.sh script  to at least note duplicates in the name or email parts of the AUTHORS file, and provide advice / help on updating .mailcap.  Perhaps there is a nice script out there that we could grab?  And then note the augmented process in the project/generate-authors.sh file and in the release checklist or whatever release documentation there is.

So where is the how-to-make-a-release documentation?

**Comment by tiborvass** [comment]
@nealmcb it's in `project/RELEASE-CHECKLIST.md`.

Not sure what the status or the purpose of this PR is anymore :(

**Comment by jfrazelle** [comment]
ping @nealmcb

**Comment by nealmcb** [comment]
Based on the conversation in my PR, we want a manual step to check for wrong addresses like @users.noreply.github.com, or other likely problems, and update the .mailcap file.

I suggest that the project/generate-authors.sh script should at least look for that sort of thing, note duplicates in the name or email parts of the AUTHORS file, and provide some sort of advice / help on updating .mailcap. Perhaps there is a nice script out there that we could grab?

The Docker release checklist is at project/RELEASE-CHECKLIST.md and it should suggest that this sort of AUTHORS update should happen at least once near the end of each release.

Shall we proceed with those items?

**Comment by tianon** [comment]
If we can come up with sane, simple logic for "detecting possible
duplicates", then I'm definitely +1.

**Comment by icecrime** [comment]
I'm +1 too! @nealmcb do you have take taking this: will you update this PR or should we close it? Thanks!

**Comment by nealmcb** [comment]
I'm continuing this discussion in the issue #10451

**Comment by tiborvass** [comment]
@nealmcb would you mind closing this PR now? That way you can reopen it once we agree on the issue you referenced.

**Comment by nealmcb** [comment]
Sure, tiborvass - thanks!  I bet that reduces clutter in the pull request list.  And I didn't realize how easy it was to just reopen it :)

**Pull request closed without merging**